test/postgresql/chunking_example.bats

```diff
@@ -4,10 +4,10 @@
 . "$BATS_TEST_DIRNAME"/env.sh >&2
 
 : ${CHUNKING_TEST_NUM_WORDS_TRAIN:=1000} ${CHUNKING_TEST_NUM_WORDS_TEST:=200}
-: ${CHUNKING_TEST_MIN_F1SCORE:=60}
+: ${CHUNKING_TEST_MIN_F1SCORE:=80}
 
 : ${CHUNKING_TEST_REUSE_NUM_WORDS_TEST:=200}
-: ${CHUNKING_TEST_REUSE_MIN_F1SCORE:=60}
+: ${CHUNKING_TEST_REUSE_MIN_F1SCORE:=80}
```

Comment:
It was PHONY! It was a mesh (<i, j> for all 0 <= i < j < N), not skip chain! Keeping that rule would drag the F1 to 53. Removing it gets us F1 = 87.